omnipaxos/tests/utils.rs

```diff
@@ -695,14 +692,9 @@ impl Entry for Value {
     type Snapshot = LatestValue;
 }
 
-fn stopsign_meta_to_value(ss: &StopSign) -> Value {
-    let v = ss
-        .metadata
-        .as_ref()
-        .expect("StopSign Metadata was None")
-        .first()
-        .expect("Empty metadata");
-    Value(*v as u64)
+fn stopsign_config_to_value(ss: &StopSign) -> Value {
+    let v = ss.next_config.configuration_id;
+    Value(v as u64)
```

Comment:
This looks a bit weird. I guess you wanted to use the ask-reply in kompact to check that the reconfiguration was decided. In that case, it is better to explicitly create the Value out of the configuration id (and comment that in the code) instead of having this function.